presenters/main_presenter.py

```diff
@@ -1,7 +1,8 @@
-class MainPresenter():
+class MainPresenter:
```

Comment:
All classes should now be so-called "new-style classes" and so should always provide a base class. In the "empty" case then classes should inherit from `object`